### PR TITLE
Add tenant rest API

### PR DESCRIPTION
**Pull request by nilasini**
Related to https://github.com/wso2/product-is/issues/8047

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/develop/restapis/keystore.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.0
+it chopenapi: 3.0.0
```
Comment (sherenewso2) [inline en/docs/develop/restapis/keystore.yaml:1]: was this change intentional?

File: en/docs/develop/restapis/tenant-management.yaml [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/tenants/{tenant-id}/owners':
+    get:
+      tags:
+        - Tenants
+      summary: Retrieve tenant's owners.
+      operationId: getOwners
+      description: |
+        Retrieve owners of the tenant which are identified by the provided tenant id.
```
Comment (sherenewso2) [inline en/docs/develop/restapis/tenant-management.yaml:179]: id --> ID

File: en/docs/develop/restapis/keystore.yaml (hunk 1 above)
Comment (nilasini) [reply en/docs/develop/restapis/keystore.yaml]: No by mistake. Removed it now.
